### PR TITLE
📝 Add features section

### LINKED ISSUE #77: 📝 Updating the description in the browser stores and README
Hello Team 👋

I think the extension changed enough since the first publication. The descriptions on the stores and in the README should be updated, including:
* [x] a description of the required permissions: what are they used for ?  (related to #75)
* [x] a part about the config available. At the 2021-04-23, these features are configurable:
  * the Dark mode
  * close the extension on a gitmoji copy
  * show the description of a gitmoji on hover or focus
* [x] how to use the extension with the shortcuts
* [x] README updated
* [ ] Descriptions in the stores updated (#79)
  * [ ] Chrome → html format is not possible, redirect to the README instead
  * [x] Firefox

### PR DESCRIPTION
**Pull request by johannchopin**
Close #77

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,15 @@ Install the extension according to your browser:
 - use <kbd>tab</kbd> and <kbd>Shift+tab</kbd> to navigate between the shortcodes and emojis
 - press <kbd>Enter</kbd> to copy
 
+## 🧩 Features
+
+- 🔍 Search for a specific gitmoji's code/emoji by name or description
+- 🌔 Dark/light mode
+- ❓ Show gitmoji's description on mouse over or focus
+- ❌ Close extension's popup after a gitmoji is saved in the clipboard
```
Comment (Lyokolux) [inline README.md:81]: This one is redundant I think

File: README.md [hunk 2]
```diff
@@ -73,6 +73,15 @@ Install the extension according to your browser:
 - use <kbd>tab</kbd> and <kbd>Shift+tab</kbd> to navigate between the shortcodes and emojis
 - press <kbd>Enter</kbd> to copy
 
+## 🧩 Features
+
+- 🔍 Search for a specific gitmoji's code/emoji by name or description
+- 🌔 Dark/light mode
+- ❓ Show gitmoji's description on mouse over or focus
```
Comment (Lyokolux) [inline README.md:80]: mention that it is available through the setting panel

File: README.md [hunk 3]
```diff
@@ -73,6 +73,15 @@ Install the extension according to your browser:
 - use <kbd>tab</kbd> and <kbd>Shift+tab</kbd> to navigate between the shortcodes and emojis
 - press <kbd>Enter</kbd> to copy
 
+## 🧩 Features
+
+- 🔍 Search for a specific gitmoji's code/emoji by name or description
+- 🌔 Dark/light mode
```
Comment (Lyokolux) [inline README.md:79]: mention that it is available through the setting panel

File: README.md (hunk 1 above)
Comment (johannchopin) [reply README.md:81]: I wanted to specify it because you can disable this behaviour in the settings page. So is there à reason to not keep it?

**Review by Lyokolux — COMMENTED** [review]
The features available in the settings should be grouped together (in a sublist ?) and explicitly described as available in the settings panel.

File: README.md (hunk 1 above)
Comment (Lyokolux) [reply README.md:81]: Done it.

**Comment by Lyokolux** [comment]
@johannchopin I handle the rest :+1: I will ping you when I will finish it

**Comment by Lyokolux** [comment]
@johannchopin please review manually and post a comment about it :pray:

**Comment by johannchopin** [comment]
I would prefer that all the features stay in the same list and contain the settings option at the end. Example:

```
- 🌔 Dark/light mode (⚙️)
```

...and then we could add a tiny section like:

> Features with the (⚙️) can be modified in the settings page

What do you think of that @Lyokolux?

**Review by Lyokolux — APPROVED** [review]

**Pull request merged**